.travis.yml

```diff
@@ -36,7 +36,6 @@ addons:
       - google-cloud-sdk
       - openssl
       - protobuf-compiler
-      - pkg-config
```

Comment:
Sorry, forgot to add another comment to this PR. This needs to be deleted too:

https://github.com/ampproject/amphtml/blob/7b1733faa56bb9524d0287840f8d8df985d63983/.travis.yml#L31-L32

I noticed because there are still failures like this on `master`:

![image](https://user-images.githubusercontent.com/26553114/82510614-045ce900-9ad9-11ea-8fc4-a88b8d020082.png)